packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.doc.tsx

```diff
@@ -1,25 +0,0 @@
-import * as React from 'react';
```

Comment:
Marking these file as deleted… does that mean we loose the git history of the file in its new place?